trace-sdk/src/androidTest/java/io/bitrise/trace/data/management/DataManagerInstrumentedTest.java

```diff
@@ -83,80 +65,24 @@ public void setUp() {
   }
 
   /**
-   * When the sending is stopped with {@link DataManager#stopSending(android.content.Context)} the
+   * When the sending is stopped with {@link DataManager#stopSending()} the
    * {@link ServiceScheduler#cancelAll()} should be called.
    */
   @Test
   public void stopSending_shouldCallCancelAll() {
-    dataManager.stopSending(context);
+    dataManager.stopSending();
 
     verify(dataManager.metricServiceScheduler, times(1)).cancelAll();
     verify(dataManager.traceServiceScheduler, times(1)).cancelAll();
   }
 
   /**
-   * Test when the Data collection in started in the DataManager, the number of active
```

Comment:
all these deletions are because there are now unit tests (cheaper vs integration) that cover the same test functions.